### PR TITLE
Capstone Project 1 - Code with Raihan

### PR DESCRIPTION
**Pull request by raihan2bd**
# Capstone project 1 💪 
### Hi There, Finally I completed my 1st capstone project by following this given [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and I called it CodeWithRaihan. This project's main theme is an online full-stack web development Bootcamp. Here is a quick preview of my project:

</p>

    ### Live Preview of my project in a mobile browser.
> <p align="center">
> </p>

### I made many changes in this pull request but some of them are:

- First of all, I create a new repo and set up linters on it then I push my main branch code on GitHub.
- Then I created a new branch and added one section at a time with responsible code.
- Day 1 I created this project home page and also add a feature tutors section in this project dynamically using javascript.
- Day 2 I created this project about page.
- Then I fixed my linter error and edit the readme file with a live link and also added preview images.

# Last but not least If you want to see my project live on GitHub pages  [click here](https://raihan2bd.github.io/code-with-raihan/)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+  </main>
+  <footer class="footer black-footer">
+    <div class="footer-block">
+      <div class="logo">
+        <img src="./images/logo-grey.png" alt="Code with Raihan"/>
+      </div>
+      <h3>&#169;2022 Code with Raihan, Inc. All Rights Reserved.</h3>
+    </div>
+  </footer>
```
Comment (ichala) [inline about.html]: - [ ] Please note that the footer background for the **_mobile version_** should be white both in the index and the about page, kindly fix it to match the design guidelines 🙏

File: index.html [hunk 2]
```diff
@@ -4,11 +4,179 @@
...
+      </ul>
+  </header> <!-- Heder end here -->
+
+  <!-- Main container start Form here -->
+  <main class="container">
+    <!-- Hero Section -->
+    <section id="hero" class="hero">
+      <h3 class="primary-color hero-greeting">"Hello Tech Lovers"</h3>
+      <h1 class="hero-title">BECOME A PROFESSIONAL<br> FULLSTACK DEVELOPER</h1>
+      <p class="hero-details">Boost your career joining our highly professional fullstack Bootcamp and start a remote international fulltime job. Through out this Bootcamp your learning path will be practical and beginner friendly.</p>
```
Comment (ichala) [inline index.html]: **_kindly fix these design issues in your index.html header to match the design🙏  :_**

- [ ] Kindly Add some space to the left for the whole content
- [ ] Kindly Increase The Size of the small title on top of the main one
- [ ] kindly update the header description style including the size and the background color, border etc ...  to match the design 
- [ ] kindly remove the button in your header and replace it with some information like shown in the design

your design : 
image


required design : 
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+    <!-- About Headline -->
+    <section id="about-headline" class="about-headline">
+      <div class="headline-block">
+      <h1 class="headline-title">Full Stack WebDevelopment<br> Bootcamp 2023</h1>
+      <p class="headline-details">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus numquam sint laborum itaque quis aut recusandae dolores corporis nesciunt pariatur esse, veritatis, aperiam deserunt perferendis dignissimos facilis! Aut, saepe pariatur?</p>
+      <h5>Please Contack us if you need any furthur information about this Bootcamp</h5>
+      <address>example@example.com</address>
+    </div>
+    </section>
```
Comment (ichala) [inline about.html:90]: **_kindly fix these design issues in your about.html header to match the design🙏  :_**

- [ ] Kindly Increase The Size of the main title to match the design 
- [ ] Kindly adjust the font weight for the email to be bolder so you match the design

your design : 
image


required design : 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+              <img src="./images/raihan.jpg" alt="Abu Raihan">
+              <h5 class="student-bootcamp">Fullstack Bootcamp 2020</h5>
+            </div>
+            <div class="student-info">
+              <h3 class="sudent-name">Abu Raihan</h3>
+              <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+            </div>
+          </li>
+      </ul>
+    </section>
```
Comment (ichala) [inline about.html]: - [ ] While you did a great job with the past event section and your design looks stunning for the cards, unfortunately following the design guidelines is part of the requirements, it's like a real-life project where you get an guideline where you have to implement it as it is, kindly updates your cards to match the design guidelines 🙏 

**_Your design :_** 
image

**_Required Design : _**
image

File: js/app.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    const offset = section.offsetTop - 200;
+    const height = section.offsetHeight;
+    const id = section.getAttribute('id');
+    if (top >= offset && top < height + offset) {
+      navLinks.forEach((link) => {
+        link.classList.remove('active');
+        document.querySelectorAll(`a[href*=${id}]`).forEach((item) => item.classList.add('active'));
+      });
+    }
+  });
```
Comment (ichala) [inline js/app.js:35]: - [ ] This piece of code is awesome I'm really impressed 👏 👏 , Great Job and keep rocking 💯

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @raihan2bd 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linters errors✔️ 
- Great Design✔️
- Good Readme File✔️

 ## Required Changes ♻️

- [ ] 🛑  As part of the project requirements, kindly re-take the video presentation and talk about a piece of code that you are proud of 🙏 
- [ ] 🛑 **_Check The comments below_**



## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @raihan2bd 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Great presentation demo 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] It will be better if you update your readme with an [updated README template](https://github.com/microverseinc/readme-template)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**